### PR TITLE
Add integration tests simulating libaries including GOV.UK Frontend

### PR DESCRIPTION
**Pull request by romaricpascal**
Add tests checking that configuration gets applied properly when libraries include GOV.UK Frontend:
- that with `@import` both GOV.UK Frontend and the library get appropriately configured by global variables
- that with `@use` the library gets appropriately affected by configuration of `pkg:govuk-frontend`
- that with `@forward` configuring the library appropriately configures both GOV.UK Frontend and a subsequent library that `@use`s GOV.UK Frontend

As we're recommending to either only `@use` or `@import` all your stylesheets, this PR does not add any tests about mixing `@use` and `@import` (we already know this may lead to duplicate output or GOV.UK Frontend getting loaded again with a blank configuration due to how Sass loads files).

### CODE REVIEW

File: tests/sass-tests/libraries/with-import/index.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+@import "pkg:govuk-frontend/index.import";
```
Comment (NickColley) [inline tests/sass-tests/libraries/with-import/index.scss:1]: I think we spoke about it before sorry but should we also have a case for 
`@import "pkg:govuk-frontend";` ?
